app/controllers/PlaylistController.js

```diff
@@ -197,4 +238,4 @@ angular.module('adminUI')
 		
 		return 0;
     }
-});
\ No newline at end of file
+});
```

Comment:
`npm test` throws an error and doesn't recognize this, that's why it needs to be `}]);`